codecparsers/jpegParser.cpp

```diff
@@ -313,6 +313,9 @@ bool Parser::nextMarker()
     const uint8_t* current = start;
     const uint8_t* match;
 
```

Comment:
hi @uartie this is not corner case in Android, it happens in any jpeg decode.
when the issue happens, currentBytePosition() = m_size = jpeg size.